src/services/news/index.js

```diff
@@ -53,6 +53,21 @@ class NewsService {
 		}
 	}
 
+	async getPermissions(news, { userId, schoolId } = {}, permissions) {
+		const retValue = [];
+		for (const key of Object.keys(permissions)) {
+			try {
+				await this.authorize(news, { userId, schoolId }, key);
```

Comment:
We should rather build this into https://github.com/schul-cloud/schulcloud-server/pull/639/files#diff-451065b2593834c206b324ac5a7db19cR30 to minimise the necessary requests (2 instead of 6)

Reply:
sounds good. tomorrow.